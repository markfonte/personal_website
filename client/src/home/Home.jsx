import React, {lazy, Suspense} from 'react';
import PropTypes from 'prop-types';
const InteractionCard = lazy(() => import('../shared/InteractionCard.jsx'));
const TicTacToeCard = lazy(() => import('./TicTacToeCard.jsx'));
const GimbleCard = lazy(() => import('./GimbleCard.jsx'));
const TechStackCard = lazy(() => import('./TechStackCard.jsx'));
const WelcomeCard = lazy(() => import('./WelcomeCard.jsx'));
const SammyCard = lazy(() => import('./SammyCard.jsx'));
import {Typography, Link, CircularProgress} from '@mui/material';
import {withStyles} from '@mui/styles';

const renderLoader = () => <CircularProgress color="secondary" />;

const getCookie = require('../shared/util/Cookies.js').getCookie;

const styles = {
  title: {
    margin: 'auto',
    maxWidth: 1200,
    textAlign: 'center',
    justifyContent: 'center',
  },
  subtitle: {
    margin: '16px',
  },
  michiganLogo: {
    margin: '0px',
  },
  web: {
    color: '#61DBFB',
  },
  twoColumnContainer: {
    display: 'flex',
    flexDirection: 'row',
    flexWrap: 'wrap',
  },
};

class Home extends React.Component {
  componentDidMount() {
    document.title = 'Home | Mark Fonte';
  }

  render() {
    const {classes} = this.props;

    const justlightLink = 'https://www.justlight.com/about';
    const teslaLink = 'https://www.tesla.com/';
    const umichLink = 'https://umich.edu/';
    const entrepreneurshipLink = 'https://innovateblue.umich.edu/academics/minor-in-entrepreneurship/';
    const webDevLink = 'https://en.wikipedia.org/wiki/Web_development';
    const androidDevLink = 'https://developer.android.com/';
    const awsLink = `https://aws.amazon.com/`;

    const lightTheme = getCookie('app_theme') === 'light_theme';
    const justlightColor = lightTheme ? 'justlight-color-dark' : 'justlight-color-light';
    const teslaColor = lightTheme ? 'tesla-color-dark' : 'tesla-color-light';
    const amazonColor = lightTheme ? 'amazon-color-dark' : 'amazon-color-light';
    const michiganColor = lightTheme ? 'michigan-color-dark' : 'michigan-color-light';
    const webColor = lightTheme ? 'web-color-dark' : 'web-color-light';
    const androidColor = lightTheme ? 'android-color-dark' : 'android-color-light';

    return (
      <div>
        <Typography className={classes.title} variant="h6">
          Senior software engineer at {' '}
          <Link href={teslaLink} className={teslaColor} underline="hover">
            Tesla
          </Link>
          {' '}· Former software lead at{' '}

          <Link href={justlightLink} className={justlightColor} underline="hover">
            JustLight
          </Link>

          {' '}· Former software engineer at{' '}
          <Link href={awsLink} className={amazonColor} underline="hover">
            Amazon Web Services
          </Link>
          {' '}· computer science in engineering degree from{' '}

          <Link href={umichLink} className={michiganColor} underline="hover">
            University of Michigan
          </Link>

          {' '}· Minor in{' '}

          <Link color="textSecondary" href={entrepreneurshipLink} underline="hover">
            entrepreneurship
          </Link>

          {' '}·{' '}

          <Link href={webDevLink} className={webColor} underline="hover">
            Web
          </Link>{' '}

          &amp;{' '}

          <Link href={androidDevLink} className={androidColor} underline="hover">
            Android
          </Link>

          {' '}dev
        </Typography>
        <Suspense fallback={renderLoader()}>
          <div className={classes.twoColumnContainer}>
            <WelcomeCard />
            <GimbleCard />
            <SammyCard />
            <TechStackCard />
          </div>
          <TicTacToeCard />
          <InteractionCard pagename="home" />
        </Suspense>
      </div>
    );
  }
}

Home.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(Home);
