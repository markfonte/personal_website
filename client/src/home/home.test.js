import React from 'react';
import Home from './home.jsx';
import renderer from 'react-test-renderer';

test('Home matches snapshot', () => {
  const component = renderer.create(
      <Home/>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
