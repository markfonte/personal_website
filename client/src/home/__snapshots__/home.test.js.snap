// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches snapshot 1`] = `
<div>
  <h6
    className="MuiTypography-root Home-title-1 MuiTypography-h6"
  >
    Incoming
     
    <a
      aria-describedby={null}
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorTextSecondary"
      href="https://amazon.jobs/en/job_categories/software-development"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      title="Go to Amazon Software Jobs page"
    >
      <i>
        Software Development Engineer
      </i>
    </a>
     
    at
     
    <a
      aria-describedby={null}
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover amazonColorLight MuiTypography-colorPrimary"
      href="https://www.amazon.com/"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      title="Go to Amazon.com"
    >
      <i>
        Amazon
      </i>
    </a>
     
    ¬∑
     
    <a
      aria-describedby={null}
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorTextSecondary"
      href="https://majors.engin.umich.edu/program/computer-science/"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      title="Go to U of M computer science page"
    >
      <i>
        Computer Science in Engineering
      </i>
    </a>
     
    degree from the
     
    <a
      aria-describedby={null}
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover michiganColorLight MuiTypography-colorPrimary"
      href="https://umich.edu/"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      title="Go to U of M website"
    >
      <i>
        University of Michigan
      </i>
    </a>
     
    ¬∑ Minor in
     
    <a
      aria-describedby={null}
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorTextSecondary"
      href="https://innovateblue.umich.edu/academics/minor-in-entrepreneurship/"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      title="Go to U of M Entrepreneurship page"
    >
      <i>
        Entrepreneurship
      </i>
    </a>
     
    ¬∑ Focus on
     
    <a
      aria-describedby={null}
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover webColorLight MuiTypography-colorPrimary"
      href="https://en.wikipedia.org/wiki/Web_development"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      title="Go to web development page"
    >
      <i>
        Web
      </i>
    </a>
     
    &
     
    <a
      aria-describedby={null}
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover androidColorLight MuiTypography-colorPrimary"
      href="https://developer.android.com/"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      title="Go to Android Developers page"
    >
      <i>
        Android
      </i>
    </a>
     
    development
  </h6>
  <div
    className="MuiPaper-root MuiCard-root card MuiPaper-elevation8 MuiPaper-rounded"
  >
    <div
      className="MuiPaper-root MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        aria-controls="welcome-content"
        aria-disabled={false}
        aria-expanded={false}
        aria-label="Welcome"
        className="MuiButtonBase-root MuiExpansionPanelSummary-root"
        id="welcome-header"
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiExpansionPanelSummary-content"
        >
          <a
            className="gone"
            href="#welcome_card"
            name="welcome_card"
          >
            Welcome Card
          </a>
          <div
            className="WelcomeCard-expansionHeaderContainer-56"
          >
            <div
              className="MuiCardHeader-root"
            >
              <div
                className="MuiCardHeader-content"
              >
                <span
                  className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                >
                  Welcome!
                </span>
                <span
                  className="MuiTypography-root MuiCardHeader-subheader MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
                >
                  Click for a little about me...
                </span>
              </div>
            </div>
            <img
              alt="Mark grad pic 1"
              className="MuiCardMedia-root WelcomeCard-welcomeImage-57 MuiCardMedia-media MuiCardMedia-img"
              src="/images/mark_grad_pic_1.jpg"
              title="Mark grad pic 1"
            />
          </div>
        </div>
        <div
          aria-disabled={false}
          aria-hidden={true}
          className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role={null}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <button
              aria-describedby={null}
              className="MuiButtonBase-root MuiIconButton-root"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              style={
                Object {
                  "padding": 0,
                }
              }
              tabIndex={0}
              title="Expand/collapse card"
              type="button"
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </button>
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root MuiCard-root card MuiPaper-elevation8 MuiPaper-rounded"
  >
    <div
      className="MuiPaper-root MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        aria-controls="built-with-table-content"
        aria-disabled={false}
        aria-expanded={false}
        aria-label="Built With Table"
        className="MuiButtonBase-root MuiExpansionPanelSummary-root"
        id="built-with-table-header"
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiExpansionPanelSummary-content"
        >
          <a
            className="gone"
            href="#built_with_table_card"
            name="built_with_table_card"
          >
            Built With Table Card
          </a>
          <div
            className="BuiltWithTableCard-expansionHeaderContainer-136"
          >
            <div
              className="MuiCardHeader-root"
            >
              <div
                className="MuiCardHeader-content"
              >
                <span
                  className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                >
                  üõ†Ô∏è Built With:
                </span>
                <span
                  className="MuiTypography-root MuiCardHeader-subheader MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
                >
                  What I used to build this website
                </span>
              </div>
            </div>
            <div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  DigitalOcean
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  Express
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  GitHub
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  Jest
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  Material-UI
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  Nginx
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  NodeJS
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  React
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  TravisCI
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
              <div
                className="MuiButtonBase-root MuiChip-root BuiltWithTableCard-tags-138 MuiChip-colorSecondary MuiChip-clickableColorSecondary MuiChip-outlined MuiChip-outlinedSecondary MuiChip-clickable"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                tabIndex={0}
              >
                <div
                  className="MuiChip-icon MuiChip-iconColorSecondary"
                />
                <span
                  className="MuiChip-label"
                >
                  Trello
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          aria-disabled={false}
          aria-hidden={true}
          className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role={null}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <button
              aria-describedby={null}
              className="MuiButtonBase-root MuiIconButton-root"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseOver={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              style={
                Object {
                  "padding": 0,
                }
              }
              tabIndex={0}
              title="Expand/collapse card"
              type="button"
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </button>
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
